### PR TITLE
Audit/Revision of Setup/Prerequisities

### PR DESCRIPTION
**Pull request by rjan90**
Initial Audit/Revision of the Setup/Prerequisities page.

Tasks that needs to be done:
- [x] Finish the revision
- [x] Spellcheck

### CODE REVIEW

**Comment by LexLuthr** [comment]
Why have we removed the bellaman_cpu variable from under optional tweaks?

**Comment by rjan90** [comment]
> Why have we removed the bellaman_cpu variable from under optional tweaks?

Removed it since it´s not a Prerequisite for initializing the Storage Provider. The Storage Provider operates and functions really well without it, and PC2 is generally faster without it.

Planning to move it to configurations, as that is more an appropriate place to add it, since it´s optional.

File: content/en/storage-providers/setup/prerequisites.md [hunk 1]
```diff
@@ -32,74 +32,43 @@ Please make sure that the following prerequites are met whether you are planning
...
 
-Nvidia RTX 3090 was used in this example. Remember to edit it with your GPU and amount of Cuda cores.
-
-### Configure parameters location
-
-For the miner to start, it will need to read and verify the Filecoin proof parameters. These can be downloaded in advance (recommended), or otherwise the init process will. Proof parameters consist of several files, which in the case of 32 GiB sectors, total **over 100 GiB**.
+### Running the storage provider on a different machine as the Lotus Node
 
-We recommend setting a custom location to store parameters and proofs parent cache -created during the first run- with:
+If you opt to run a the `lotus-miner` on a different machine then the Lotus Node, set:
```
Comment (TippyFlitsUK) [inline content/en/storage-providers/setup/prerequisites.md:61]:
```suggestion
If you opt to run the `lotus-miner` on a different machine than the Lotus Node, set:
```

**Review by TippyFlitsUK — COMMENTED** [review]
Looking good!!

**Review by TippyFlitsUK — APPROVED** [review]

**Pull request merged**